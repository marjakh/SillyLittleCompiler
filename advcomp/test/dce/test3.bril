@main {
  a: int = const 4;
  a: int = const 5;
  print a;
}
