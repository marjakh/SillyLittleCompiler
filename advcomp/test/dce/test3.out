@main {
  a: int = const 5;
  print a;
}
